gravity.sh

```diff
@@ -92,20 +97,6 @@ gravity_collapse() {
 		done < ${adListDefault}
 		echo " done!"
 	fi
-
-	# Create the pihole resource directory if it doesn't exist.  Future files will be stored here
-	if [[ -d ${piholeDir} ]];then
-        # Temporary hack to allow non-root access to pihole directory
-        # Will update later, needed for existing installs, new installs should
-        # create this directory as non-root
-        chmod 777 ${piholeDir}
```

Comment:
Okay, if you click on "Files Changed" at the top of the screen, it should bring up a list of file differences, from there click on the line number of the file you are commenting on and leave a line note so we know exactly which line you are referring to. Thanks!